src/components/ArticlePage/CommentBox.js

```diff
@@ -0,0 +1,34 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+const CommentBox = (props) => {
+  const { comment } = props;
+  return (
+    <Fragment>
+      <div className="comment-box">
+        <div className="author-container">
+          <img src={comment.author.avatarUrl} alt="author" />
+          <div>
+            <span id="author-name">{comment.author.fullName}</span>
+            <div>
+              <span id="author-date">{comment.date}</span>
+            </div>
+          </div>
+        </div>
+
+        <p>{comment.body}</p>
+        <div className="options-container">
+          <i className="fa fa-thumbs-o-up" />
```

Comment:
I think the javascript to have this changing color/class onClick should be included.

Reply:
I've added logic state logic to manage the icon class. I think the onClick for this should fire off a reducer that will change the state and then the icon's class, so I can't write that yet as it's beyond the scope of this task.

Reply:
Update: I have not added all the click handlers for the buttons. They all now work.